view/src/__tests__/FunctionListing.test.jsx

```diff
@@ -0,0 +1,52 @@
+// Unit tests of FunctionListing component
+// Author: Lukas Petr
+
+import { screen, render } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import '@testing-library/jest-dom';
+
+import FunctionListing from '../components/FunctionListing';
+
+it('headline should be visible', () => {
+  render(
+    <FunctionListing
+      headline="Funkce"
```

Comment:
Let's stick with English
```suggestion
      headline="Function"
```
same for 2 more occurrences below